### PR TITLE
Transfer Flow and Type Updates

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#15292

*Description of changes:*
* Implements Transfer Types
* Updated Transfer Flow and Validations(More to come here in subsequent ticket)
* Updated unit testing
* Relocation of Declared Value (Additional Validation to come in subsequent ticket)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by owillborn — APPROVED** [review]
LGTM, lots of work done here! Going to have some conflicts with my most recent PR no biggie though :)

**Comment by cameron-eyds** [comment]
> LGTM, lots of work done here! Going to have some conflicts with my most recent PR no biggie though :)

Shouldn't be too bad! Your work is mostly additive, maybe we get lucky.

File: ppr-ui/src/composables/mhrInformation/useMhrInformation.ts [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
 import { useActions, useGetters } from 'vuex-composition-helpers'
 import { readonly, ref } from '@vue/composition-api'
-import { ActionTypes, ApiHomeTenancyTypes, HomeTenancyTypes, UIRegistrationTypes } from '@/enums'
+import {
+  ActionTypes,
+  ApiHomeTenancyTypes,
+  ApiTransferTypes,
+  HomeTenancyTypes,
+  UIRegistrationTypes,
+  UiTransferTypes
```
Comment (doug-lovett) [inline ppr-ui/src/composables/mhrInformation/useMhrInformation.ts]: "Ui" intentional?

Comment (cameron-eyds) [reply ppr-ui/src/composables/mhrInformation/useMhrInformation.ts]: o good catch, will fix!

**Review by doug-lovett — APPROVED** [review]
Looks great, just the one comment which I see you already responded to.

**Pull request merged**